src/config/builder.rs

```diff
@@ -375,7 +392,7 @@ mod tests {
     /// should ideally be able to fix so that the original hash passes!
     fn version_hash_match() {
         assert_eq!(
-            "14def8ff43fe0255b3234a7c3d7488379a119b7dbcf311c77ad308a83173d92c",
+            "e71a87b3e4c761176a9f90aa94387e26383113554a119b1cbaaa25d439f25464",
```

Comment:
This is intentional here since we are modifying the global config struct, but cc/ @leebenson @001wwang in case there are any repercussions of this.

Reply:
I think the only effect would be that existing pipelines (like [the ones](https://app.datadoghq.com/pipelines?from_ts=1651256265783&to_ts=1651259865783) in Obs Pipelines Dev org) would get a new version reported (and reflected in UI) when rebooted even without user-made config changes. Could be a bit unexpected but otherwise should be ok.